packages/blitz-next/src/index-browser.tsx

```diff
@@ -37,6 +45,9 @@ const buildWithBlitz = <TPlugins extends readonly ClientPlugin<object>[]>(plugin
         setTimeout(() => {
           document.documentElement.classList.add("blitz-first-render-complete")
         })
+        document.addEventListener("blitz:session-created", async () => {
+          await Promise.all(globalThis.__BLITZ_onSessionCreated())
+        })
```

Comment:
this needs to go in the `blitz` package because it's a global event that all plugins should use, not just nextjs apps.

It can go in the `setupBlitzClient` function, but just only run it if `isClient === true`